prow/config/values.yaml

```diff
@@ -19,14 +19,14 @@ ingress:
 
 crier:
   scrapeMetrics: false
-  image: gcr.io/k8s-prow/crier:v20210422-d12e80af3e
+  image: gcr.io/k8s-prow/crier:v20221121-dfec9ce18d
```

Comment:
Same comment here, maybe we can raise a PR Just for bumping images version. Just in case something breaks it's gonna be easier to debug (and worst case revert)

Reply:
If I don't bump the version in this PR, then everything will be broken until the other PR is merged.  Shouldn't we aim to make PRs idempotent?

Reply:
Ohhh gotcha! Well in the case let's keep them this way